<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-OG1O4LU" failures="0" tests="6" name="com.testcases.DarticanAutomation.Currency" time="-1677846016.000" errors="1" timestamp="2023-03-03T17:26:20 PKT" skipped="0">
  <testcase classname="com.testcases.DarticanAutomation.Currency" name="mouseover" time="5.755"/>
  <system-out/>
  <testcase classname="com.testcases.DarticanAutomation.Currency" name="columnname" time="-1677846400.000">
    <error message="
Can inject only one of &lt;ITestContext, XmlTest&gt; into a @AfterClass annotated columnname.
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Can inject only one of <ITestContext, XmlTest> into a @AfterClass annotated columnname.
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection
at org.testng.internal.Parameters.checkParameterTypes(Parameters.java:437)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:351)
at org.testng.internal.Parameters.createParameters(Parameters.java:708)
at org.testng.internal.Parameters.createConfigurationParameters(Parameters.java:166)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:299)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:130)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- columnname -->
  <system-out/>
  <testcase classname="com.testcases.DarticanAutomation.Currency" name="ValidationErrors" time="60.336"/>
  <system-out/>
  <testcase classname="com.testcases.DarticanAutomation.Currency" name="AddCurrency" time="181.225"/>
  <system-out/>
  <testcase classname="com.testcases.DarticanAutomation.Currency" name="EditCurrency" time="171.646"/>
  <system-out/>
  <testcase classname="com.testcases.DarticanAutomation.Currency" name="Deletecurrency" time="70.830"/>
  <system-out/>
</testsuite> <!-- com.testcases.DarticanAutomation.Currency -->
